src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SNI/SNINpHandle.cs

```diff
@@ -38,15 +37,14 @@ internal sealed class SNINpHandle : SNIPhysicalHandle
         private int _bufferSize = TdsEnums.DEFAULT_LOGIN_PACKET_SIZE;
         private readonly Guid _connectionId = Guid.NewGuid();
 
-        public SNINpHandle(string serverName, string pipeName, long timerExpire, object callbackObject)
+        public SNINpHandle(string serverName, string pipeName, long timerExpire)
         {
             long scopeID = SqlClientEventSource.Log.TrySNIScopeEnterEvent("<sc.SNI.SNINpHandle.SNINpHandle |SNI|INFO|SCOPE> Constructor");
             SqlClientEventSource.Log.TrySNITraceEvent("<sc.SNI.SNINpHandle.SNINpHandle |SNI|INFO> Constructor. server name = {0}, pipe name = {1}", serverName, pipeName);
             try
             {
                 _sendSync = new object();
                 _targetServer = serverName;
-                _callbackObject = callbackObject;
```

Comment:
Why is this removed?

Reply:
It's not needed, never used.

Reply:
If that is the case, should we also remove it from its caller:
private SNINpHandle CreateNpHandle(DataSource details, long timerExpire, object callbackObject, bool parallel) ?

Reply:
Done as per Davoud's above comment.